### DISCUSSION TITLE
How do you disable the slider on mobile?

### DISCUSSION THREAD

**Discussion by notflip** [nolimits4web/swiper#6949] [Q&A] [answered]
For mobile users I would just like to disable the slider so all slides are positioned underneath each other, has anyone found a way of doing this?

**Comment by Arhell** [comment, marked as answer]
Use https://swiperjs.com/swiper-api#method-swiper-disable on https://swiperjs.com/swiper-api#param-breakpoints

**Reply by notflip** [reply]
Thanks @Arhell

**Reply by melodyxpot** [reply]
Thanks @Arhell , this answer is very helpful. Cheers.

**Reply by VIPStephan** [reply]
How exactly would disable() be used on the breakpoints parameter? 🤔 Can you give an example?

**Reply by vsxed** [reply]
> How exactly would disable() be used on the breakpoints parameter? 🤔 Can you give an example?

You can simply use the param `enabled: bool`

e.g.

```js
...
enabled: false,
breakpoints: {
   480: {
       enabled: true,
   }
}
```

This would initially disable the slider and if the breakpoint is `>=480px`, it will be enabled.
